tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -97,3 +101,18 @@ def cleanup(self):
 
     def executor_browser(self):
         return ExecutorBrowser, {"webdriver_url": self.server.url}
+
+def run_info_extras(**kwargs):
+    osReleaseCommand = "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').ReleaseId"
+    osBuildCommand = "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').BuildLabEx"
+    try:
+        os_release = call("powershell.exe", osReleaseCommand).strip()
+        os_build = call("powershell.exe", osBuildCommand).strip()
+    except (subprocess.CalledProcessError, OSError):
+        return {}
+
+    rv = {
+        "os_build": os_build,
+        "os_release": os_release}
```

Comment:
With the `{` on a new line, I'd rather the `}` be on one too.

Reply:
I've fixed this up to be formatted the same as the firefox file (open brace on same line, in-line key values, close brace on same line)